main.cpp

```diff
@@ -270,10 +280,11 @@ extern "C" void* ThreadCrawler(void* data)
             res.nHeight = 0;
             res.strClientV = "";
             bool getaddr = res.ourLastSuccess + 86400 < now;
-            res.fGood = TestNode(res.service, res.nBanTime, res.nClientV, res.strClientV, res.nHeight, getaddr ? &addr : NULL);
+            res.fGood = TestNode(res.service, res.grapheneVersion, res.electrsVersion, res.capdVersion, res.nBanTime, res.nClientV, res.strClientV, res.nHeight, getaddr ? &addr : NULL);
+            //    if (res.fGood) printf("\nClient: %s, graphene: %lu\n", res.strClientV.c_str(), (long unsigned int) res.grapheneVersion);
```

Comment:
to be removed?

Reply:
its a pretty useful printout for debugging

Reply:
I wonder that once we get this done including this source code into the main BU repo would made this kind of scenario more convenient by using LOG/LOGA macros

Reply:
yes, we should definitely convert all these ugly printouts to LOG messages when we convert. I don't think its worth it to move the LOG infra over here though.